integration/cdi/src/main/java/org/apache/cxf/cdi/CdiBusBean.java

```diff
@@ -63,6 +65,9 @@ public String getName() {
     @Override
     public ExtensionManagerBus create(final CreationalContext< ExtensionManagerBus > ctx) {
         final ExtensionManagerBus instance = injectionTarget.produce(ctx);
+        if ("true".equals(SystemPropertyAction.getProperty("org.apache.cxf.cdi.useCdiUnwrapper", "true"))) {
```

Comment:
@johnament pretty minor, before merging could you please rename property `org.apache.cxf.cdi.useCdiUnwrapper` to `org.apache.cxf.cdi.unwrap.proxies`  (or similar) so it kind of explains its purpose.

Reply:
Sure